proto/push/service.proto

```diff
@@ -84,18 +84,18 @@ message ListPushesResponse {
 message DeletePushRequest {
   reserved 1;
   string id = 2 [(google.api.field_behavior) = REQUIRED];
-  DeletePushCommand command = 3 [deprecated = true];
-  string environment_id = 4;
+  reserved 3;
+  string environment_id = 4 [(google.api.field_behavior) = REQUIRED];
 }
 
 message DeletePushResponse {}
 
 message UpdatePushRequest {
   reserved 1;
   string id = 2 [(google.api.field_behavior) = REQUIRED];
-  AddPushTagsCommand add_push_tags_command = 3 [deprecated = true];
-  DeletePushTagsCommand delete_push_tags_command = 4 [deprecated = true];
-  RenamePushCommand rename_push_command = 5 [deprecated = true];
+  reserved 3;
+  reserved 4;
+  reserved 5;
```

Comment:
For reserved fields, we put the field info as a comment for future reference.

```suggestion
  reserved 3; // AddPushTagsCommand add_push_tags_command
  reserved 4; // DeletePushTagsCommand delete_push_tags_command
  reserved 5; // RenamePushCommand rename_push_command
```

Also, you didn't delete the [command.proto](https://github.com/bucketeer-io/bucketeer/blob/main/proto/push/command.proto) file